passive/JavaDisclosure.js

```diff
@@ -0,0 +1,29 @@
+//Passive scan for Java error messages containing sensitive information (CWE-209)
+
+function scan(ps, msg, src) {
+    var alertRisk = 2
+    var alertConfidence = 3
+    var alertTitle = 'Java stack trace disclosure (or similar) - investigation required (script)'
+    var alertDesc = 'Java stack trace disclosure (or similar) was found'
+    var alertSolution = 'Investigate Java stack trace disclosures found in the response, remove or mask as required'
+    var cweId = 209
+    var wascId = 0
+
+    var re = /springframework|\.java|rootBeanClass/i
+
+    var url = msg.getRequestHeader().getURI().toString()
```

Comment:
Should be done later (when raising the alert) to void creating a new string unnecessarily.

Reply:
Fixed 👍